src/package/plugins/slate-alignment-plugin/AlignmentCenterButton.js

```diff
@@ -2,11 +2,12 @@ import React from 'react'
 import classnames from 'classnames'
 import FontAwesome from 'react-fontawesome'
 
+import { Button} from '../../components/button'
 import { alignmentMarkStrategy, hasMark, getMark } from './AlignmentUtils'
 
 
 const AlignmentCenterButton = ({ state, onChange, changeState, className, style }) => (
-  <button
+  <Button
```

Comment:
@abobwhite Can you add the `type` prop to all the plugins button to provide a way to change the type of button component, since now we can edit the button's type?